DungeonAndDragonsRollStats/src/main/app/Driver.java

```diff
@@ -10,32 +10,38 @@
 import java.util.Scanner;
 import java.util.stream.Collectors;
 
-import enums.Ability;
-import enums.CharacterClass;
-import enums.CharacterRace;
-import models.Player;
-import models.Player.PlayerBuilder;
+import main.enums.Ability;
+import main.enums.CharacterClass;
+import main.enums.CharacterRace;
+import main.enums.DiceFaceCount;
+import main.models.Player;
+import main.models.Player.PlayerBuilder;
+import main.services.DiceRollerService;
+import main.services.DiceRollerServiceImpl;
+import main.services.DiceSumService;
+import main.services.DiceSumServiceImpl;
 
 public class Driver {
 
 	private static final int STAT_ROLL_DICE_COUNT = 4;

```

Comment:
this value is specifically for rolling your ability stats, which is where you roll 4 and remove the lowest roll. So for this value, it should be constant. This will likely be put in it's own class/enum for number of dice to roll.